test/support/index.ts

```diff
@@ -26,6 +26,11 @@ Cypress.on('uncaught:exception', (err) => {
   return !err.message.includes('getBoundingClientRect');
 });
 
+// Extends timeout counter to 30000 ms
+Cypress.config('defaultCommandTimeout', 30000);
```

Comment:
shouldn't we change the config here? https://github.com/mitre/heimdall2/blob/master/cypress.config.ts

Reply:
Changed as requested.